### ISSUE TITLE
Add subject terms to collection-level metadata records

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#425]
### Descriptive Summary

In order to enable collections to be matched when searching, the collection level records need to contain subject terms.  Collections will need to be subject analyzed and/or subject terms assigned to the collection's MARC finding aids will be copied into the DAMS records.

### Background 

This approach is much simpler and more straight forward than pulling subject terms assigned to works with in the collection to be pulled up to the collection level.

### Acceptance Criteria

This is what done looks like:

* [ ] Each collection record has relevant subject terms

### Related Work 

This is a prerequisite to Create Matching Collections Box #224

**Comment by rmjaffe** [comment]
@tmariamora @katedundon As I go about assigning collection-level subject terms, I plan to refer to the finding aids on OAC.  Are LCSH subject terms generally found in the collection details, or might they appear in other sections of the finding aids?

**Comment by katedundon** [comment]
@rmjaffe you are correct- subject terms in OAC finding aids are in Collection Details under Indexing Terms (or sometimes the heading Subjects and Indexing Terms). They can also be found in the collection level catalog record.

**Comment by rmjaffe** [comment]
@NedHenry I just went into the Harry Mayo collection to add subject terms to the collection level metadata, but it doesn't look like there's the option.  When clicking [Edit Collection](https://digitalcollections.library.ucsc.edu/dashboard/collections/2n49t173w/edit?locale=en), the Title, Abstract/Summary and Thumbnail properties are exposed.  And when I clicked on Additional Fields, as few more options revealed themselves, including a Keyword property, but none of our controlled subject properties:  subjectName, subjectPlace, subjectTopic, subjectTitle.

Can these properties be added to the collection editing form?  Would that be an easy change?  If so, I can make a ticket for the work.  If it isn't, I'm thinking we and the rest of the team should talk about what it would mean to use the keyword property instead.
